containers/datalab/content/run.sh

```diff
@@ -54,6 +54,21 @@ check_tmp_directory() {
     echo "The /tmp directory is writable"
 }
 
+# Reinstall the parts of pydatalab from /content/pydatalab, which is where it gets live-mounted.
+function reinstall_pydatalab() {
+  PYDATALAB=/content/pydatalab
+  echo "Reinstalling pydatalab from ${PYDATALAB}"
+  pip uninstall -y datalab
+  pip3 uninstall -y datalab
+  pip install --force-reinstall --no-cache-dir ${PYDATALAB}
+  pip install --force-reinstall ${PYDATALAB}/solutionbox/image_classification/.
+  pip install --force-reinstall ${PYDATALAB}/solutionbox/structured_data/.
+  pip3 install --force-reinstall --no-cache-dir ${PYDATALAB}
+  pip3 install --force-reinstall ${PYDATALAB}/solutionbox/image_classification/.
```

Comment:
This was directly copied from the Dockerfile, where it does the same thing. chmeyers suggested keeping them in in both places, and it will start working once DataFlow supports py3.